package/yast2-network.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Thu Nov 21 07:13:28 UTC 2019 - Michal Filka <mfilka@suse.com>
+
+- fate#319639
```

Comment:
I would prefer to use the bug number instead of the fate, so, if you wanted to mention it then add both.